### PR TITLE
test: Fix zora fork tests

### PR DESCRIPTION
**Pull request by arr00**
This PR fixes the Zora fork tests such that they actually run on a full party instance

### CODE REVIEW

File: test/proposals/ListOnZoraProposalForked.t.sol [hunk 1]
```diff
@@ -208,36 +165,126 @@ contract ListOnZoraProposalForkedTest is TestUtils {
...
-        _expectEmit0();
+        _expectEmit3();
         emit ZoraAuctionSold(proposalData.token, proposalData.tokenId);
-        assertTrue(proposal.executeListOnZora(executeParams).length == 0);
-        assertEq(address(proposal).balance, proposalData.listPrice);
+        assertTrue(executeProposal(proposalId, proposal, progressData).length == 0);
+        assertEq(address(party).balance, proposalData.listPrice);
+    }
+
+    /// MARK: Helpers
```
Comment (0xble) [inline test/proposals/ListOnZoraProposalForked.t.sol:232]: Delete this?

**Review by 0xble — APPROVED** [review]
Super helpful, thanks

File: test/proposals/ListOnZoraProposalForked.t.sol (hunk 1 above)
Comment (arr00) [reply test/proposals/ListOnZoraProposalForked.t.sol:232]: i kinda like it. If you are against i can delete

Comment (0xble) [reply test/proposals/ListOnZoraProposalForked.t.sol:232]: Not opinionated on it, if you like it we can keep it

**Pull request merged**